### PR TITLE
cleanup: Docker compat m1, dep updates, script fix

### PR DESCRIPTION
**Pull request by exrhizo**
Updated docs

Updated deps for `client-api` and `client-api-react` and `cra-test` and lerna

Small bugfix in `extract-dist.sh`

### CODE REVIEW

**Comment by exrhizo** [comment]
@lmeyerov there is an issue with my release to npm where the `client-api-react` doesn't import because of an issue with how I did the release and the dist folder for that package, so I want to merge these and make a new minor revision.

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
-FROM node:16.13.0-slim as base
+FROM node:16.13.0-alpine as base
+WORKDIR /opt/graphistry-js
+COPY lerna.json package.json package-lock.json ./
+RUN apk add --no-cache python3 make g++
```
Comment (lmeyerov) [inline Dockerfile:4]: why do we need need py3/g++/etc for pure node apps?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,7 @@
-FROM node:16.13.0-slim as base
+FROM node:16.13.0-alpine as base
```
Comment (lmeyerov) [inline Dockerfile:1]: i've historically hit random networking issues with alpine (musc etc), recommend we stick w/ slim?

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+WORKDIR /opt/graphistry-js
+COPY lerna.json package.json package-lock.json ./
```
Comment (lmeyerov) [inline Dockerfile:3]: duplicate of lines below?

**Review by lmeyerov — CHANGES_REQUESTED** [review]
made a quick pass, basic stuff, see comments

File: Dockerfile (hunk 2 above)
Comment (exrhizo) [reply Dockerfile]: reverted

File: Dockerfile (hunk 1 above)
Comment (exrhizo) [reply Dockerfile]: explained in PR comment

File: tools/jsdocs.sh [hunk 4]
```diff
@@ -9,7 +9,7 @@ VERSION=$(cat projects/client-api/package.json | jq -r .version)
 rm -rf docs-build/jsdocs
 mkdir -p docs-build/jsdocs
 docker run --rm \
-    --entrypoint=/bin/bash \
+    --entrypoint=/bin/sh \
```
Comment (lmeyerov) [inline tools/jsdocs.sh:12]: guessing these b/c you had the alpine switch at first? but probably ok

**Review by lmeyerov — APPROVED** [review]
lgtm

**Pull request merged**